docker-compose.prod.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.1'
+services:
+  db:
+    image: postgres
+    environment:
+      POSTGRES_DB: circuitverse_production
+      POSTGRES_PASSWORD: postgres
+  sidekiq:
+    depends_on:
+      - db
+      - redis
+    build: .
+    command: bundle exec sidekiq -e production -q default -q mailers -d -L tmp/sidekiq.log
+    volumes:
+      - .:/circuitverse
+      - ./config/database.docker.yml:/circuitverse/config/database.yml
+    environment:
+      REDIS_URL: "redis://redis:6379/0"
+  web:
+    build: .
+    entrypoint: /circuitverse/bin/docker_run
```

Comment:
this docker run command is for development environment only, as it run db:seed file

Reply:
@thiefie, you need to make changes to `/circuitverse/bin/docker_run` file.